services/core-web/src/components/Forms/ExplosivesPermit/ExplosivesPermitForm.tsx

```diff
@@ -220,282 +236,289 @@ export const ExplosivesPermitForm: FC<ExplosivesPermitFormProps &
             cancelText="No"
             onConfirm={props.closeModal}
           >
-            <Button className="full-mobile">
-              Cancel
-            </Button>
+            <Button className="full-mobile">Cancel</Button>
           </Popconfirm>
-          <Button disabled={isAmend} type="primary" onClick={(e) => handleOpenAddExplosivesPermitModal()}>
+          <Button
+            disabled={isAmend}
+            type="primary"
+            onClick={() => handleOpenAddExplosivesPermitModal()}
```

Comment:
and got rid of the unused function arg here, also because eslint wanted me to.

The rest was just eslint cleaning up the formatting on its own.